msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -412,6 +414,12 @@ Update app manifest (previously called Teams app manifest) with the `composeExte
   "composeExtensions": [
     {
       "composeExtensionType": "apiBased",
+      "authorization": {
+        "authType": "apiSecretServiceAuth ",
+        "apiSecretServiceAuthConfiguration": {
+            "apiSecretRegistrationId": ""sCVBX2udSXEtxo97behM1ReO8pJc4MdA"
```

Comment:
Can you change this exaample to a GUID? The generated api keys are actually guids. Developers might be confused to see that the value they get is very different from the example in the documentation.